modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -189,6 +189,94 @@ definition {
 			actionName = "ActionName");
 	}
 
+	@description = "LPS-145665 - Verify it's possible to add an object entry with the trigger On Order Status Update"
+	@priority = "4"
+	test CanCreateActionAddObjectEntryWithOnOrderStatusUpdate {
+		task ("Given an action in the Commerce Order triggered with Custom Object.") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 171861",
+				objectName = "CustomObject171861",
+				pluralLabelName = "Custom Objects 171861");
+
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object 171861");
+
+			ObjectAdmin.goToFieldsTab();
```

Comment:
You can omit these steps since you are adding the Object field via API. No need to navigate through the UI.